etc/help.txt

```diff
@@ -195,15 +195,26 @@ Tcl commands (Available in GUI or Batch console or Batch script):
                               : Uses alternate bitstream generation configuration files
 </openfpga>
 
----------------
---- Program ---
----------------
+------------------
+--- Programmer ---
+------------------
    programmer <command> 
-     command                      :fpga_config, flash, fpga_status, list_devices
-     fpga_config <bitstream_file> : Program the device via JTAG
-     flash <bitstream_file>       : Program the flash via JTAG
-     fpga_status                  : Get the status of the device
-     list_devices                 : List all available devices
+     command                             :fpga_config, flash, fpga_status, list_device, list_cable
+     fpga_config -c <cable name/index> ?-d <device_index>? <bitstream_file> : Program the FPGA device via JTAG
+         -c <cable_name/index>    : Specify the cable name or index
+         -d <device_index>        : Specify the device index
+         <bitstream_file>         : Bitstream file to program
+     flash -c <cable name/index> ?-d <device_index>? ?-o <operations>? <bitstream_file> : Program the Flash via JTAG
+         -c <cable_name/index>    : Specify the cable name or index
+         -d <device_index>        : Specify the device index
+         -o <operations>          : Specify the operations to perform on the flash (erase|blankcheck|program|verify)
```

Comment:
Please remove the flash operations that are not supported yet.

Reply:
Removed the unsupported operations help